rebar.config

```diff
@@ -61,6 +61,12 @@
   {short, [{erl_opts, [{d, 'EQC_SHORT_TEST'}]}]},
   {long, [{erl_opts, [{d, 'EQC_LONG_TEST'}]}]},
   {prod, [{relx, [{dev_mode, false}]}]},
+  {deb, 
+	  [{relx, 
+		  [{dev_mode, false},
+		  {overlay_vars, "rel/vars/deb.config"}]},
+	  {cuttlefish,
```

Comment:
No sorry this would break backwards compatibility changing it isn't really an option.